src/components/Banner/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+
+import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage"
+
+import DCLLogo from "../../images/dcl-logo.svg"
+import GenesisBanner from "../../images/genesis-banner.png"
+import WorldBanner from "../../images/worlds-banner.png"
+import WorldsLogo from "../../images/worlds-logo.svg"
+import { Close } from "../Icon/Close"
+
+import "./index.css"
+
+type BannerProps = {
+  type: "places" | "worlds"
```

Comment:
If we add the enum, this would be 

```suggestion
  type: CategoryCountTargetOptions.PLACES | CategoryCountTargetOptions.WORLDS
```